README.md

```diff
@@ -1,7 +1,9 @@
 [![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)
+[![Build Status](https://travis-ci.org/GoogleCloudPlatform/appengine-plugins-core.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/appengine-plugins-core)
+
 # Google App Engine Plugins Core Library
 
-This is the client Java library to manage App Engine Java applications. It should be used by any application that performs App Engine Java application management. For example, the Maven, Gradle and Eclipse App Engine plugins, custom user tools, etc.
+This is the client Java library to manage App Engine Java applications. It should be used by any Java application that performs App Engine Java application management. For example, the Maven, Gradle and Eclipse App Engine plugins, custom user tools, etc.
```

Comment:
I've always thought of this as the Cloud SDK-based App Engine Java SDK. But that direction might have changed while I was out. You could argue that there is already the App Engine Admin API Java client lib which does the same things that this one does. Since we're also not planning to implement staging and all other goodies that gcloud does here, it's probably OK to reword.